.gitlab-ci.yml

```diff
@@ -69,11 +69,18 @@ variables:
     matrix:
       - DISTRO: debian
         RELEASE: [buster, bullseye, bookworm]
-        ARCH: [amd64, armhf, arm64]
+        ARCH: [amd64]
       - DISTRO: ubuntu
         RELEASE: [focal, jammy]
         ARCH: [amd64, armhf, arm64]
 
+.mender-dist-packages-image-matrix-arm:
+  parallel:
+    matrix:
+      - DISTRO: debian
+        RELEASE: [buster, bullseye, bookworm]
+        ARCH: [armhf, arm64]
```

Comment:
This looks really promising! Great idea introducing ARM CI runners.

The only jobs that we need to run on actual ARM machines are:
```
    matrix:
      - DISTRO: debian
        RELEASE: [buster, bullseye, bookworm]
        ARCH: [armhf]
```

The rest of the ARM combinations, for example `ubuntu armhf`, `ubuntu arm64` and `debian arm64` are using the regular cross-compile toolchain from Debian upstream and they need to run in native x86_64 (we could experiment on having these also doing native builds on ARM, but let's keep that for a different iteration).